src/scripts/configure.sh

```diff
@@ -18,8 +18,3 @@ if [ "$PARAM_AWS_CLI_CONFIG_PROFILE_REGION" = "1" ]; then
     aws configure set region "$PARAM_AWS_CLI_REGION" \
         --profile "$PARAM_AWS_CLI_PROFILE_NAME"
 fi
-
-if [ -n "$PARAM_AWS_CLI_ROLE_ARN" ]; then
-    aws configure set role_arn "$PARAM_AWS_CLI_ROLE_ARN" \
```

Comment:
@KyleTryon - I removed this line and added just the `aws configure set......`  to the role-arn-setup script below